### PR TITLE
Add an installer name variable for generated archive

### PR DESCRIPTION
**Pull request by salabon77mk**

### CODE REVIEW

File: lib/gitlab/ci/templates/pipeline/GradleInstall4JPipeline.yml [hunk 1]
```diff
@@ -30,7 +31,7 @@ before_script:
   variables:
     SNAPSHOT: "-Psnapshot" # default to snapshot
   artifacts:
-    name: installers
+    name: $INSTALLER_NAME installers
     expire_in: 10 days
```
Comment (salabon77mk) [inline lib/gitlab/ci/templates/pipeline/GradleInstall4JPipeline.yml:37]: I'm open to re-arranging the installer name here as I'm unsure if INSTALL_NAME is blank, then there will be an extra white space character that may mess with people who use command line tools.

One thing I could do is make the default value of INSTALLER_NAME be "installers" and let any downstream users rename it from there.

Comment (CorriganRJ) [reply lib/gitlab/ci/templates/pipeline/GradleInstall4JPipeline.yml:37]: Can you assign INSTALLER_NAME the default value of "installers" in the variables section at the top of the file and add a description so it's easier for folks to override the value in manual pipelines?

```
variables:
   INSTALLER_NAME:
      value: "installers"
      description: "The name of the installer artifacts that can be downloaded after the job completes"
```

**Comment by CorriganRJ** [comment]
I changed the target branch from master to `release/1.2` since `Master` is only updated when I'm going to cut a new release branch with breaking changes. I'll update the repo settings so that `release/1.2` shows up as the target branch for future PRs as well

File: lib/gitlab/ci/templates/pipeline/GradleInstall4JPipeline.yml (hunk 1 above)
Comment (salabon77mk) [reply lib/gitlab/ci/templates/pipeline/GradleInstall4JPipeline.yml:37]: Should be done :)

**Review by CorriganRJ — APPROVED** [review]

**Pull request merged**